### PR TITLE
[DOCFIX] Include new collectInfo features into the doc

### PR DESCRIPTION
**Pull request by jiacheliu3**

### CODE REVIEW

File: docs/en/operation/Troubleshooting.md [hunk 1]
```diff
@@ -116,26 +116,81 @@ like -Daws.access.key=XXX, DO NOT share the collected tarball with anybody unles
...
+"2020-01-03 12:10:11"
+"2020-01-03 12:10"
+"20/01/03 12:10:11"
+"20/01/03 12:10"
+2020-01-03T12:10:11.874+0800
+2020-01-03T12:10:11
+2020-01-03T12:10
+```
+
+1. `--start-time <datetime>` specifis a datetime before with the log files can be ignored.
```
Comment (gpang) [inline docs/en/operation/Troubleshooting.md:191]:
```suggestion
1. `--start-time <datetime>` specifies a datetime before with the log files can be ignored.
```

**Review by gpang — CHANGES_REQUESTED** [review]
@jiacheliu3 just a minor comment

**Review by gpang — APPROVED** [review]
LGTM

**Comment by gpang** [comment]
alluxio-bot, merge this please

**Comment by jiacheliu3** [comment]
alluxio-bot, cherry-pick this to branch-2.3 please

**Pull request merged**